### DISCUSSION TITLE
Guidance in multiple scRNA seq data

### DISCUSSION THREAD

**Discussion by Lutfi-Hodali** [satijalab/seurat#5439] [Q&A]
Hi, 
I am new to the scRNA seq world and I'm trying to analyze published data for my own studies.
In this article 10.15252/embj.2020107333, the data they provide appears to be divided based on patients, meaning that they provide matrices and barcodes for each patient, and only one features file.
I'll gladly appreciate the help in directing me towards the type of analysis that I should use, is it like the Vignette, "introduction to scRNA-seq integration" or do I need to do something else.
Thank you!
Lutfi-Hodali

**Comment by samuel-marsh** [comment]
Hi,

The type of analysis you will need to perform is unfortunately going to be highly experiment and analysis-goal specific.  Some experiments may do better without integration while others may need it.  The same goes for your preference of using the basic workflow vs. scTransform workflow.  I would recommend taking look at many of the vignettes and what the published study did, and what others studies do, as well as consulting with bioinformatician (if possible) before deciding what the best workflow is going to be for your study.

Best,
Sam

**Reply by Lutfi-Hodali** [reply]
Sorry for the late reply. Thank you!!
Lutfi
